fme.js

```diff
@@ -0,0 +1,54 @@
+      window.rdoEvents = {
+        general: [
```

Comment:
What's the thinking behind changing this name? I thought 'freeRoam' was more specific (I've heard them referred to as free-roam events a lot), and more future-proof.

Reply:
I have the FMEs on the Collectors Map / RDO Map and it was just easier for me to copy the list across without making changes, only a little thing and also since they're all named Free Mode Events in the actual game files too with the roles ones tagged as such fme_role_greatest_bounty_hunter for Day of Reckoning for example.

Obviously not a big deal just just was easier to keep on top of both json files and make sure they both match, obviously the Map version is different as the Map versions have the game code event name, is was just so I know both are correct and when I need to make changes I can do both lists at the same time: https://jeanropke.github.io/RDR2CollectorsMap/data/fme.json